index.html

```diff
@@ -121,12 +121,13 @@ <h4>Apply Now!</h4>
         <p><a type="button" href="https://ddf.volunteermeet.org/general_volunteer_application" target="_">Apply Here!</a></p>
     </section>   
     <section class="hidden" id="contact-section">
-      <main class="content-container">
+      <div class="content-container">
         <h3>Get In Touch</h3>
         <p>Email us at <a href="mailto:t4rj-contact@digitaldefensefund.org">t4rj@digitaldefensefund.org</a> to chat!</p> 
+        <h4>Questions or concerns regarding tech?</h4>
```

Comment:
I think it'd be great to add a blurb here about what the tech doula program is. Here's some text...

> Talk to a tech doula! Just as a birth or abortion doula can offer holistic support before during and after a pregnancy, T4RJ Tech Doulas offer individualized and compassionate support for tech needs and questions. Tech doesn't have to be intimidating, paternalistic or scary.
> 
> You can request support through the form below. If we can’t fulfill your request, we will point you in the direction of other resources that may be helpful. We look forward to connecting with you!

Reply:
Thank you for the text suggestion!